doc/configuration.md

```diff
@@ -97,3 +97,17 @@ This flag periodically prunes the consumed tx_out table. So it allows to query f
 without having to maintain the whole tx_out table. Deletes to `tx_out` are propagated to `ma_tx_out`
 through foreign keys. If this is set once, then it must be always set on following executions of
 db-sync. Failure to do this can result in crashed and db-sync currently has no way to detect it.
+
+### --bootstrap
```

Comment:
Is there a reason for the name `bootstrap`? (Is it blockchain/cardano related)

Just when I read "bootstrap" is doesn't make me think it has something to do with the faster syncing of UTxO.

maybe `fast-utxo-sync`, `deferred-utxo-insertion` !?!?! I'm not 100% just that "bootstrap" feels a little vague :)
``

Reply:
I used bootstrap since this process "instantly" loads the `tx_out` table from the ledger state instead of going through the phase of syncing.

Reply:
ah I see, could it be `bootstrap_tx_out`?

Reply:
Good idea. Renamed to `bootstrap-tx-out` so it's more clear and matches the other flags `prune-tx-out` `consumed-tx-out`

Reply:
perfect thanking you 😄